### PR TITLE
Code sample - Spring JdbcTemplate JDBC ADB UCP

### PR DESCRIPTION
**Pull request by juarezjuniorgithub**
Code sample - Spring JdbcTemplate JDBC ADB UCP
Signed-off-by: Juarez Barbosa juarez.barbosa@oracle.com

### CODE REVIEW

**Comment by juarezjuniorgithub** [comment]
@Kuassim @jeandelavarene the adjustments were performed as required - indentation fixed. 
Please review, thanks.

File: java/jdbc-spring-jdbctemplate-oracle/src/main/java/com/oracle/jdbc/template/EmployeeMessages.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.oracle.jdbc.template;
+
+public enum EmployeeMessages {
+
+  EMPLOYEE_CREATION_FAILED(
+      "Employee creation failed: "), EMPLOYEE_UPDATE_FAILED(
```
Comment (jeandelavarene) [inline java/jdbc-spring-jdbctemplate-oracle/src/main/java/com/oracle/jdbc/template/EmployeeMessages.java:27]: Please fix the format

File: java/jdbc-spring-jdbctemplate-oracle/src/main/java/com/oracle/jdbc/template/EmployeeSqlStatements.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.oracle.jdbc.template;
+
+public enum EmployeeSqlStatements {
+
+  FIND_ALL_EMPLOYEES("SELECT * FROM employee"), FIND_EMPLOYEE_BY_ID(
+      "SELECT * FROM employee WHERE id = :id"), CREATE_NEW_EMPLOYEE(
```
Comment (jeandelavarene) [inline java/jdbc-spring-jdbctemplate-oracle/src/main/java/com/oracle/jdbc/template/EmployeeSqlStatements.java:27]: Please fix the format

**Review by Kuassim — APPROVED** [review]

**Review by Kuassim — APPROVED** [review]

File: java/jdbc-spring-jdbctemplate-oracle/src/main/java/com/oracle/jdbc/template/EmployeeMessages.java (hunk 1 above)
Comment (juarezjuniorgithub) [reply java/jdbc-spring-jdbctemplate-oracle/src/main/java/com/oracle/jdbc/template/EmployeeMessages.java:27]: Indentation is double-checked for all classes. Please note and remember the enforced format for String and enum data types as expected, per the IDE conventions we configured together. @Kuassim please review, thanks

File: java/jdbc-spring-jdbctemplate-oracle/src/main/java/com/oracle/jdbc/template/EmployeeSqlStatements.java (hunk 2 above)
Comment (juarezjuniorgithub) [reply java/jdbc-spring-jdbctemplate-oracle/src/main/java/com/oracle/jdbc/template/EmployeeSqlStatements.java:27]: The indentation is double-checked for all classes. Please note and remember the enforced format for String and enum data types as expected, per the IDE conventions we configured together. @Kuassim please review, thanks

**Pull request closed without merging**